src/chain/validators-impl.spec.ts

```diff
@@ -154,12 +190,48 @@ describe('strict #isBoolean()', () => {
     await isBoolean.run(context, 0, meta);
     await isBoolean.run(context, 'true', meta);
     await isBoolean.run(context, 'false', meta);
+    // The two below are not passing because we are using a custom validator which does take the raw value and do not pass it through `toString`
```

Comment:
nit

```suggestion
    // The two below are not passing because we are using a custom validator which takes the raw value and does not pass it through `toString`
```